x-pack/plugins/uptime/public/lib/helper/test_helpers.ts

```diff
@@ -62,6 +62,10 @@ export function mockReduxHooks(response?: any) {
   jest.spyOn(redux, 'useSelector').mockReturnValue(response);
 }
 
+export function mockDispatch() {
```

Comment:
Mocking useSelector was causing issues for me, since this component called on two different selectors, so I am only mocking dispatch. Mocking useSelector is less critical when using the default app state within a mock redux provider, as we are in the new rtl custom render.